benchmark/large/ingestion/load_tpcds_sf100.sql

```diff
@@ -0,0 +1,99 @@
+CALL dsdgen(sf=100);
+EXPORT DATABASE 'duckdb_benchmark_data/tpcds_parquet_sf100' (FORMAT PARQUET);
+EXPORT DATABASE 'duckdb_benchmark_data/tpcds_csv_sf100' (FORMAT CSV);
+
+create view call_center_csv as select * from read_csv('duckdb_benchmark_data/tpcds_csv/call_center.csv');
+create view call_center_parquet as select * from read_parquet('duckdb_benchmark_data/tpcds_parquet/call_center.parquet');
+create view call_center_native as select * from call_center;
+
+create view household_demographics_csv as select * from read_csv('duckdb_benchmark_data/tpcds_csv/household_demographics.csv');
+create view household_demographics_parquet as select * from read_parquet('duckdb_benchmark_data/tpcds_parquet/household_demographics.parquet');
+create view household_demographics_native as select * from household_demographics;
+
+create view store_returns_csv as select * from read_csv('duckdb_benchmark_data/tpcds_csv/store_returns.csv');
```

Comment:
we don't need all of these views, Only csv/parquet/native views for the `store_sales` and `inventory` tables are needed, since those are the only tables we ingest